src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -505,6 +529,85 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
     }
   }
 
+  override def checkUserIsInExistingGoogleGroup(groupEmail: String, userEmail: String): Int = {
+    val directoryService = getDirectoryManager(getDelegatedCredentialForAdminUser)
+
+    val memberGetRequest = directoryService.members.get(groupEmail, userEmail)
+
+    Try(executeGoogleRequest(memberGetRequest)) match {
+      case Failure(f) => {
+        val errorcode = f.asInstanceOf[GoogleJsonResponseException].getDetails.getCode
+        if (errorcode != 404) { // log errors other than 404 (404 means group doesn't exist or member not in group)
+          val message = f.asInstanceOf[GoogleJsonResponseException].getDetails.getMessage
+          logger.warn(s"Error checking if $userEmail is member of $groupEmail, error code $errorcode, message: $message")
+        }
+        errorcode
+      }
+      case Success(f) => {
+        200 // 200 is a successful response code
+      }
+    }
+  }
+
+  override def deleteGoogleGroup(groupEmail: String): Unit = {
+    val directoryService = getDirectoryManager(getDelegatedCredentialForAdminUser)
+    val deleteGroupRequest = directoryService.groups.delete(groupEmail)
+    Try(executeGoogleRequest(deleteGroupRequest)) match {
+      case Failure(_) => {
+        logger.warn(s"Failed to delete group $groupEmail")
+      }
+      case Success(_) => {}
```

Comment:
Could be simplified:
```
      case Failure(_) => logger.warn(s"Failed to delete group $groupEmail")
      case Success(_) =>
```

Reply:
I think I can even remove the whole `case Success(_) =>` line